exasol/toolbox/util/dependencies/poetry_dependencies.py

```diff
@@ -111,44 +116,34 @@ def direct_dependencies(
     ) -> OrderedDict[str, dict[NormalizedPackageStr, Package]]:
         dependencies = OrderedDict()
         for group in self.groups:
-            command = (
+            proc = run_command(
```

Comment:
Simplified and unified the existing calls to `subprocess.run()` by using the centralized function `run_command()` instead.